soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -252,6 +253,35 @@ static std::map<std::string, CosmeticOption> cosmeticOptions = {
+    COSMETIC_OPTION("Key.WaterBossBase",            "Water Boss Key",           COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 0,   255), false, true, false),
+    COSMETIC_OPTION("Key.SpiritSmallBase",          "Spirit Small Key",         COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.SpiritIcon",               "Spirit Key Gem/Icon",      COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.SpiritBossBase",           "Spirit Boss Key",          COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 0,   255), false, true, false),
+    COSMETIC_OPTION("Key.ShadowSmallBase",          "Shadow Small Key",         COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.ShadowIcon",               "Shadow Key Gem/Icon",      COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.ShadowBossBase",           "Shadow Boss Key",          COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 0,   255), false, true, false),
+    COSMETIC_OPTION("Key.WellSmallBase",            "Well Small Key",           COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.WellIcon",                 "Well Key Icon",            COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.FortSmallBase",            "Fortress Small Key",       COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.FortIcon",                 "Fortress Key Icon",        COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.GTGSmallBase",             "GTG Small Key",            COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.GTGIcon",                  "GTG Key Icon",             COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.GanonsSmallBase",          "Ganon's Small Key",        COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    COSMETIC_OPTION("Key.GanonsIcon",               "Ganon's Key Gem/Icon",     COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
+    COSMETIC_OPTION("Key.GanonsBossBase",           "Ganon's Boss Key",         COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 0,   255), false, true, false),
+    //COSMETIC_OPTION("Key.ChestSmallBase",           "Chest Game Key",           COSMETICS_GROUP_KEY,          ColorRGBA8(255, 255, 255, 255), false, true, false),
+    //COSMETIC_OPTION("Key.ChestIcon",                "Chest Game Key Icon",      COSMETICS_GROUP_KEY,          ColorRGBA8(255, 0,   0,   255), false, true, true),
```

Comment:
I know they're just comments for now but I'd like to see these renamed to `Key.ChestGameSmallBase` or `Key.TreasureGameSmallBase` etc to make it clearer they're for the treasure chest minigame.